Source/Core/Core/PowerPC/Jit64/Jit_SystemRegisters.cpp

```diff
@@ -410,6 +410,12 @@ void Jit64::mfspr(UGeckoInstruction inst)
   case SPR_PMC2:
   case SPR_PMC3:
   case SPR_PMC4:
+  case SPR_UPMC1:
+  case SPR_UPMC2:
+  case SPR_UPMC3:
+  case SPR_UPMC4:
+  case SPR_IABR:
+  case SPR_DABR:
```

Comment:
This needs to be changed in JitArm64 as well. (You may also want to add SPR_PMC1-4 to JitArm64's list of fallbacks for this instruction. They appear to be missing...)